voc/database_storage.go

```diff
@@ -32,4 +32,5 @@ var DatabaseStorageType = []string{"DatabaseStorage", "Storage", "Resource"}
 // DatabaseStorage is an entity in our Cloud ontology. describes the actual database or a table in a database
 type DatabaseStorage struct {
 	*Storage
+	PublicAccess bool `json:"publicAccess"`
```

Comment:
For me it is ok. But we should discuss it internally how we want to do that. E.g., maybe we should an http endpoint here as well.

Reply:
That is changed back. See the comment of @oxisto.